docs/content/docs/components/cdn/_index.md

```diff
@@ -3,26 +3,35 @@ title: "CDN"
 weight: 3
 ---
 
-<b style="color: red">⚠ Do not activate CDN log processing in production yet. It's in active development.
-Be sure that config flag 'enableLogProcessing' is set to false</b>
-
 ## What's CDN
+CDN is a service dedicated to receive and store CDS's logs. In a near future it will also be able to manage artifacts and cache used by your jobs. 
+
+CDN stores the list of all known log or artifact items in a Postgres database and communicate with storage backends to store the content of this items.
+This backends are call units and there are two types of units in CDN:
+
+* Buffer Unit: to store incoming job's logs and artifact, this units are designed to be fast for read/write operations but with a limited capacity.
 
-CDN is a service dedicated to receive and retrieve logs. 
+* Storage Unit: to store complete job's logs and artifact.
 
-A CDN instance is started with scope Project and Run using token.
+When logs or file are received by CDN, it will first store this items in its buffer. Then when the item is fully received, it will be moved to one of configured storage unit.
+If CDN is setup with multiple storage units, each unit will periodically checks for missing item and sync this items from other units.
```

Comment:
```suggestion
When the logs or file are received by CDN, it will first store these items in its buffer. Then, when the item is fully received, it will be moved to one of the configured storage units.
If the CDN is configured with multiple storage units, each unit periodically checks for missing items and synchronizes those items from other units.
```